docs/sources/tempo/operations/monitor/set-up-monitoring.md

```diff
@@ -12,45 +12,45 @@ If you don't have a cluster available, you can use the linked documentation to s
 
 To set up monitoring, you need to:
 
-* Use Grafana Agent Flow or Grafana Alloy to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/latest/setup/set-up-test-app/).
-* Update your Grafana Agent Flow configuration to scrape metrics to monitor for your Tempo data.
+* Use Grafana Alloy collector to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
+* Update your collector configuration to scrape metrics to monitor for your Tempo data.
 
-This procedure assumes that you have set up Tempo [using the Helm chart](https://grafana.com/docs/tempo/latest/setup/helm-chart/) and with [Grafana Agent](https://grafana.com/docs/agent/latest/flow/) or [Grafana Alloy](https://grafana.com/docs/alloy/latest/).
+This procedure assumes that you have set up Tempo [using the Helm chart](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/helm-chart/) with [Grafana Alloy](https://grafana.com/docs/alloy/<TEMPO_VERSION>/set-up/install/).
 
-{{< docs/shared source="alloy" lookup="agent-deprecation.md" version="next" >}}
-
-The steps outlined below use the Grafana Agent Flow configurations described in [Set up a test application for a Tempo cluster](https://grafana.com/docs/tempo/latest/setup/set-up-test-app/).
+The steps outlined below use the Grafana Alloy configurations described in [Set up a test application for a Tempo cluster](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
```

Comment:
We've been defining `Grafana Alloy` in the metadata descriptions, H1s and at the first mention. Then all other mentions are just `Alloy`. I see a few places here where it's full name and short name, example line 69 and again in lines 87/88.  There are more like this.  I'm guessing it's an artefact of converting Grafana Agent to Grafana Alloy and then converting Flow to just Alloy.

It's not a fixed rule, just mentioning it. It's good with the full name too.

Reply:
You're exactly right that this is a left over from the conversion from Agent to Alloy. I'll merge your suggestions and then do a review of the doc for consistently.

Reply:
```suggestion
The steps outlined below use the Alloy configurations described in [Set up a test application for a Tempo cluster](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
```